fairlearn/datasets/synthetic.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+import numpy as np
+from sklearn.datasets import make_classification
+
+
+def make_synthetic_dataset(seed=12345):
+    """Create a synthetic dataset.
+
+    Parameters
+    ----------
+    seed : int, default=12345
+        The random number generator seed to use.
+
+    Returns
+    -------
+    (X, y, gender) : tuple of numpy.ndarray
+        X : ndarray
+            The generated samples.
+        y : ndarray
+            Labels for the binary classification.
+        gender : ndarray
+            The sensitive feature label.
+    """
+    rng = np.random.RandomState(seed=seed)
+
+    classification_kwargs = {
+        'n_features': 20,
+        'n_informative': 4,
```

Comment:
Can these be input args with these defaults?

Reply:
Sure!